### ISSUE TITLE
Implement Semantic HTML 5 layout structure

### ISSUE THREAD

**Issue by mlabsch** [mlabsch/oxideshop_ce#3]
Currently, all templates are structured with simple div elements. This doesn't provide any semantic meaning for search engines and/ or screen readers which is kind of a barrier for SEO and accessability. To enhance this features, HTML 5 should be implemented. For backward compatibility, html5shim should be implemented accordingly.
